docs/content/0.1.0/docs/getting-started/namespaces.md

```diff
@@ -0,0 +1,41 @@
+---
+weight: 15
+---
+
+# Namespaces
+
+All nodes in DataJunction exist within a namespace. Node names are dot separated alpha-numeric elements. The leading elements
+identify the namespace where the node exists. Nodes that do not include any dots in the name are automatically
+defined in the `default` namespace.
+
+| Node Name           | Namespace    |
+|---------------------|--------------|
+| roads.demo.repairs  | roads.demo   |
+| finance.revenue     | finance      |
+| hr.people.employees | hr.people    |
+| customer            | default      |
+
+Since namespaces are inferred directly from the node name, creating a node in a particular namespace simply requires prefixing
+the node name with the namespace. In other words, creating a node named `roads.demo.repairs` will automatically create the node in the
+`roads.demo` namespace.
```

Comment:
Should we tell users here that they can't create nodes in namespaces that don't exist yet?